src/pages/settings/Payments/paymentPropTypes.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import CONST from '../../../CONST';
+
+/** Array of bank account objects */
+const bankAccountListPropTypes = PropTypes.arrayOf(PropTypes.shape({
+    /** The name of the institution (bank of america, etc) */
+    addressName: PropTypes.string,
+
+    /** The masked bank account number */
+    accountNumber: PropTypes.string,
+
+    /** The bankAccountID in the bankAccounts db */
+    bankAccountID: PropTypes.number,
+
+    /** The bank account type */
+    type: PropTypes.string,
+}));
+
+/** Array of card objects */
+const cardListPropTypes = PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Also only discovered one usage for these so they do not need to be in their own file.